flutter_screen_recording_web/lib/flutter_screen_recording_web.dart

```diff
@@ -16,35 +19,107 @@ class WebFlutterScreenRecording extends FlutterScreenRecordingPlatform {
   MediaRecorder mediaRecorder;
   Blob recordedChunks;
   String mimeType;
+  Completer<String> _onStopCompleter;
+  Function(String result) onStop;
+
+  AudioContext _audioContext;
+  MediaStreamAudioDestinationNode _audioDestinationNode;
+
+  //id, source
+  Map<String, MediaStreamAudioSourceNode> _audioSourceNodes;
+
+  WebFlutterScreenRecording();
 
   static registerWith(Registrar registrar) {
     FlutterScreenRecordingPlatform.instance = WebFlutterScreenRecording();
   }
 
   @override
-  Future<bool> startRecordScreen(String name) async {
-    return _record(name, true, false);
+  Future<bool> startRecordScreen(
+    String name, {
+    Function(String) onStop,
+  }) async {
+    return _record(
```

Comment:
Because the function is `startRecordScreen`. i think _record should be `_recording` or `_recordingScreen` or `_startRecordScreen`